src/builder/pod/container.rs

```diff
@@ -359,6 +359,7 @@ mod tests {
                 ]
                 .into(),
             ),
+            claims: None,
```

Comment:
For tests i would default:
```suggestion
            ..ResourceRequirements::default()
```

Reply:
Makes sense